### PR TITLE
Info icon for branch

### PR DESCRIPTION
**Pull request by Sarkar**
usability

### CODE REVIEW

File: BranchMonsterFactory/ViewControllers/BranchInfoViewController.m [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+#import "BranchInfoViewController.h"
+
+@interface BranchInfoViewController ()
+
+@end
+
+@implementation BranchInfoViewController
+
+- (void)viewDidLoad {
```
Comment (shortstuffsushi) [inline BranchMonsterFactory/ViewControllers/BranchInfoViewController.m]: Remove unused stubs please.

File: BranchMonsterFactory/ViewControllers/MonsterViewerViewController.m [hunk 2]
```diff
@@ -303,6 +306,13 @@ - (IBAction)cmdFacebookClick:(id)sender {
     
 }
 
+/*- (IBAction)cmdInfoClick:(id)sender {
```
Comment (shortstuffsushi) [inline BranchMonsterFactory/ViewControllers/MonsterViewerViewController.m]: Also remove commented out methods.

**Comment by shortstuffsushi** [comment]
Are you planning to do submodule separately?

**Comment by shortstuffsushi** [comment]
The info button gets a little bit clipped at some screen sizes:

screen shot 2015-05-20 at 9 27 02 am

Also, this is what I was talking about doing for making the buttons "consistent:"
screen shot 2015-05-20 at 9 30 16 am

**Comment by Sarkar** [comment]
@shortstuffsushi submodule separately!

**Comment by shortstuffsushi** [comment]
:+1:

**Pull request merged**